ios/FluentUI.Demo/FluentUI.Demo/ColoredPillBackgroundView.swift

```diff
@@ -34,6 +34,11 @@ class ColoredPillBackgroundView: UIView {
         updateBackgroundColor()
     }
 
+    override func didMoveToWindow() {
```

Comment:
and this is because, there isn't a listener machanism for demo controllers? should we have todos?

Reply:
Yeah, I can add a quick todo. It's "good enough" for our demo needs, and a real fix would be more expensive.